packages/functional-tests/tests/syncV3/signUp.spec.ts

```diff
@@ -43,7 +43,7 @@ test.describe('Firefox Desktop Sync v3 sign up', () => {
     expect(await connectAnotherDevice.fxaConnected.isVisible()).toBeTruthy();
   });
 
-  test.skip('coppa disabled', async ({ target }) => {
```

Comment:
I'm not sure when we don't display coppa for sync user signups? Feels like something we can always show? @LZoog

Reply:
I wonder if we did this when we first introduced it? Agreed it seems fine to remove now. BTW what was (possibly) causing the flakiness?

Reply:
Not really sure, I think it might be cascading issues from other tests.